### ISSUE TITLE
Bevy Event/Callback Example

### ISSUE THREAD

**Issue by Trequetrum** [djeedai/bevy_tweening#18]
The docs show 4 features front and center for this crate. 

----

The first three are all well covered by the examples you have, but 

> Raise a Bevy event or invoke a callback when an tween completed

is not. At least one example per major feature would be nice :)

**Comment by djeedai** [comment]
There's an event example in [`examples/sequence.rs`](https://github.com/djeedai/bevy_tweening/blob/454bcd00ef209709b52be8a655bb613e3ba02e7c/examples/sequence.rs#L123).

**Comment by SUPERCILEX** [comment]
This issue can probably be closed.

**Comment by djeedai** [comment]
I'm conflicted. @Trequetrum has a point here, that feature is highlighted but there's no discoverable way to find an example for it. On the other hand there do is an example, if one manages to find it.

**Comment by SUPERCILEX** [comment]
Fair point. A use case I've run into for events is despawning upon completion. Maybe an example could be created around that?

**Comment by djeedai** [comment]
Yes, because that's also a feature that has been requested, so showing how to achieve it is valuable.
